.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!--
+Thanks for submitting a change to ownCloud!
+
+This is the bug tracker for the oCIS component. Find other components at https://github.com/owncloud/core/blob/master/.github/CONTRIBUTING.md#guidelines
+
+For fixing potential security issues please see https://owncloud.org/security/
+
+To make it possible for us to get your change reviewed and merged please carefully fill out the requested information below.
+
+Please note that any kind of change needs first be submitted to the master branch which holds the next version of ownCloud.
```

Comment:
```suggestion
Please note that any kind of change needs first be submitted to the master branch which holds the next version of OCIS.
```

Maybe change this to avoid confusion with ownCloud10?

Reply:
thanks for giving it another pair of eyes @phil-davis